tests/unit_tests/test_unit.py

```
@@ -268,7 +268,9 @@ def test_version():
 def test_n_workers(monkeypatch):
     """Check that n_workers is correct without a GPU."""
     monkeypatch.setattr("torch.cuda.is_available", lambda: False)
-    cpu_fun = lambda x: ["foo"] * 31
+
+    def cpu_fun(x):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `cpu_fun`

(ANN202)